package.json

```diff
@@ -8,25 +8,25 @@
     "@progress/kendo-date-math": "^1.5.1",
     "@progress/kendo-drawing": "^1.9.4",
     "@progress/kendo-licensing": "^1.1.4",
-    "@progress/kendo-react-buttons": "^4.5.0",
-    "@progress/kendo-react-charts": "^4.5.0",
-    "@progress/kendo-react-data-tools": "^4.5.0",
-    "@progress/kendo-react-dateinputs": "^4.5.0",
-    "@progress/kendo-react-dialogs": "^4.5.0",
-    "@progress/kendo-react-dropdowns": "^4.5.0",
-    "@progress/kendo-react-editor": "^4.5.0",
-    "@progress/kendo-react-excel-export": "^4.5.0",
-    "@progress/kendo-react-form": "^4.5.0",
-    "@progress/kendo-react-grid": "^4.5.0",
-    "@progress/kendo-react-indicators": "^4.5.0",
-    "@progress/kendo-react-inputs": "^4.5.0",
-    "@progress/kendo-react-intl": "^4.5.0",
-    "@progress/kendo-react-layout": "^4.5.0",
-    "@progress/kendo-react-pdf": "^4.5.0",
-    "@progress/kendo-react-progressbars": "^4.5.0",
-    "@progress/kendo-react-scheduler": "^4.6.0",
-    "@progress/kendo-react-treeview": "^4.5.0",
-    "@progress/kendo-react-upload": "^4.5.0",
+    "@progress/kendo-react-buttons": "5.5.0",
+    "@progress/kendo-react-charts": "5.5.0",
+    "@progress/kendo-react-data-tools": "5.5.0",
+    "@progress/kendo-react-dateinputs": "5.5.0",
+    "@progress/kendo-react-dialogs": "5.5.0",
+    "@progress/kendo-react-dropdowns": "5.5.0",
+    "@progress/kendo-react-editor": "5.5.0",
+    "@progress/kendo-react-excel-export": "5.5.0",
+    "@progress/kendo-react-form": "5.5.0",
+    "@progress/kendo-react-grid": "5.5.0",
+    "@progress/kendo-react-indicators": "5.5.0",
+    "@progress/kendo-react-inputs": "5.5.0",
+    "@progress/kendo-react-intl": "5.5.0",
+    "@progress/kendo-react-layout": "5.5.0",
+    "@progress/kendo-react-pdf": "5.5.0",
+    "@progress/kendo-react-progressbars": "5.5.0",
+    "@progress/kendo-react-scheduler": "5.5.0",
+    "@progress/kendo-react-treeview": "5.5.0",
+    "@progress/kendo-react-upload": "5.5.0",
     "@progress/kendo-theme-default": "^4.36.0",
```

Comment:
Version `4` of `@progress/kendo-theme-default` it not compatible with version `5` of the `@progress/kendo-react-*` packages.